configuration/micro_kernel_trait.rst

```diff
@@ -298,6 +298,10 @@ Now it looks like this::
         }
     }
 
+.. deprecated:: 6.4
+
+    Annotations are deprecated since Symfony 6.4, use attributes instead.
```

Comment:
Instead of adding this deprecation block, I'd remove the `doctrine/annotations` dependency from the command below. We should not recommend to install that package anymore fro new applications.